JetNews/app/src/debug/java/com/example/jetnews/PostPreviewParameterProvider.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2021 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.jetnews
+
+import androidx.compose.ui.tooling.preview.PreviewParameterProvider
+import com.example.jetnews.data.posts.impl.post1
+import com.example.jetnews.data.posts.impl.post2
+import com.example.jetnews.data.posts.impl.post3
+import com.example.jetnews.data.posts.impl.post4
+import com.example.jetnews.data.posts.impl.post5
+import com.example.jetnews.model.Post
+
+class PostPreviewParameterProvider : PreviewParameterProvider<Post> {
```

Comment:
Add comments about what this does and information about the feature, and also maybe link to the tooling docs?

Reply:
what's the best practice for this? Should the file be moved to something like a `previews` package?

Reply:
I moved the class to a `model` package. That way, the preview class is in the same directory as the Model class that it creates. This to me seems a logical organization for it.

I added a comment explaining the usage. There's no Tooling doc to point to (yet), as there's no documentation of this feature anywhere. I will add that to the Tooling doc when I find the time for it!

Reply:
SG! Thank you